app/services/activities/import_from_csv.rb

```diff
@@ -238,6 +261,20 @@ def convert_gdi(gdi)
         )
       end
 
+      def convert_policy_marker(policy_marker)
+        return "not_assessed" if policy_marker.blank?
+
+        policy_markers_iati_codes_to_enum(policy_marker)
+      end
+      alias convert_policy_marker_gender convert_policy_marker
+      alias convert_policy_marker_climate_change_adaptation convert_policy_marker
+      alias convert_policy_marker_climate_change_mitigation convert_policy_marker
+      alias convert_policy_marker_biodiversity convert_policy_marker
+      alias convert_policy_marker_desertification convert_policy_marker
+      alias convert_policy_marker_disability convert_policy_marker
+      alias convert_policy_marker_disaster_risk_reduction convert_policy_marker
+      alias convert_policy_marker_nutrition convert_policy_marker
```

Comment:
I like how this is organised -- avoiding quite a lot of boilerplate for defining the same conversion process for all these fields but using the allowed list to make it safe.

Reply:
Ta 👍 I did consider using `method_missing`, but this seemed clearer